### PR TITLE
fix windows path d:

### PR DESCRIPTION
**Pull request by hliebscher**
some typos d: in path...

### CODE REVIEW

**Comment by SoftFever** [comment]
@hliebscher 
These codes are not used.

File: resources/images/OrcaSlicer_gray.svg
Comment (SoftFever) [inline resources/images/OrcaSlicer_gray.svg:1]: What has been changed?

File: src/slic3r/GUI/GCodeViewer.cpp [hunk 1]
```diff
@@ -1320,7 +1320,7 @@ static void debug_calibration_output_thumbnail(const ThumbnailData& thumbnail_da
         }
     }
 
-    image.SaveFile("D:/calibrate.png", wxBITMAP_TYPE_PNG);
+    image.SaveFile("calibrate.png", wxBITMAP_TYPE_PNG);
```
Comment (SoftFever) [inline src/slic3r/GUI/GCodeViewer.cpp:1323]: These are obsoleted codes.
It is not used.

File: src/slic3r/GUI/Gizmos/GLGizmoText.cpp [hunk 2]
```diff
@@ -71,7 +71,7 @@ class Polygon_3D
 // for debug
 void export_regions_to_svg(const Point &point, const Polygons &polylines)
 {
-    std::string path = "D:/svg_profiles/text_poly.svg";
+    std::string path = "/svg_profiles/text_poly.svg";
```
Comment (SoftFever) [inline src/slic3r/GUI/Gizmos/GLGizmoText.cpp:74]: These are debug codes.
It is not used.

**Comment by hliebscher** [comment]
i can close this now windows build works ... i have change the path only for me 

cmake .. -G "Visual Studio 16 2019" -DBBL_RELEASE_TO_PUBLIC=1 -DCMAKE_PREFIX_PATH="%DEPS%/usr/local" -DCMAKE_INSTALL_PREFIX="./OrcaSlicer" -DCMAKE_BUILD_TYPE=Release -DWIN10SDK_PATH="C:/Program Files (x86)/Windows Kits/10"

**Pull request closed without merging**